src/renderer/components/dock/terminal.ts

```diff
@@ -49,8 +49,8 @@ export class Terminal {
   private xterm: XTerm | null = new XTerm({
     cursorBlink: true,
     cursorStyle: "bar",
-    fontSize: 13,
-    fontFamily: "RobotoMono",
+    fontSize: UserStore.getInstance().terminalFontSize || defaultFontSize,
```

Comment:
`UserStore.getInstance().terminalFontSize` should always be set, even to the default value.

Reply:
Also, I would say this is a good oportunity to do some DI changes, but I have already done some of it, so lets do that later.